src/core/sys/posix/sched.d

```diff
@@ -135,30 +135,7 @@ version( Posix )
 int sched_yield();
 */
 
-version( linux )
-{
-    int sched_yield();
-}
-else version( OSX )
-{
-    int sched_yield();
-}
-else version( FreeBSD )
-{
-    int sched_yield();
-}
-else version (Solaris)
-{
-    int sched_yield();
-}
-else version (Android)
-{
-    int sched_yield();
-}
-else
-{
-    static assert(false, "Unsupported platform");
-}
+int sched_yield();
```

Comment:
[Defined this way in POSIX standard](http://pubs.opengroup.org/onlinepubs/9699919799/functions/sched_yield.html).